/data/serve/ghettorest/cgi/test.sh: command substitution: line 14: unexpected EOF while looking for matching ``'
/data/serve/ghettorest/cgi/test.sh: command substitution: line 15: syntax error: unexpected end of file
/data/serve/ghettorest/cgi/test.sh: line 13: ${*[0]}: bad substitution
[Thu Feb 20 00:48:24 2014] [error] [client 172.16.0.154] file permissions deny server execution: /data/serve/ghettorest/cgi/gapi.pl
[Thu Feb 20 00:51:48 2014] [error] [client 172.16.0.154] script not found or unable to stat: /data/serve/ghettorest
[Thu Feb 20 00:55:11 2014] [notice] caught SIGTERM, shutting down
[Thu Feb 20 00:55:17 2014] [alert] httpd: Could not determine the server's fully qualified domain name, using dnpcor4rtr00.int.ghetto.sh for ServerName
[Thu Feb 20 00:55:17 2014] [notice] Initializing etag from /data/serve/ghettoapi/logs/etag-state
[Thu Feb 20 00:55:17 2014] [notice] Apache configured -- resuming normal operations
[Thu Feb 20 00:55:17 2014] [notice] Accept mutex: sysvsem (Default: sysvsem)
[Thu Feb 20 00:56:22 2014] [notice] SIGHUP received.  Attempting to restart
[Thu Feb 20 00:56:22 2014] [alert] httpd: Could not determine the server's fully qualified domain name, using dnpcor4rtr00.int.ghetto.sh for ServerName
[Thu Feb 20 00:56:22 2014] [notice] Initializing etag from /data/serve/ghettoapi/logs/etag-state
[Thu Feb 20 00:56:22 2014] [notice] Apache configured -- resuming normal operations
[Thu Feb 20 00:56:22 2014] [notice] Accept mutex: sysvsem (Default: sysvsem)
[Thu Feb 20 03:40:15 2014] [error] [client 172.16.0.178] File does not exist: /data/serve/ghettoapi/htdocs/favicon.ico
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 5, near "%ENV{QUERY_STRING"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 03:43:14 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 5, near "%ENV{"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 03:43:22 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 5, near "%ENV{"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 03:43:22 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 5, near "%ENV{"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 03:43:28 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 10, near ") {"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 03:46:25 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 12, near ")
  print"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 03:48:00 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
[Thu Feb 20 04:00:09 2014] [error] [client 127.0.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking_in&date=1392894009&uname=OpenBSD&uptime= 4:00AM  up 18 days, 19:37, 3 users, load averages: 0.10, 0.14, 0.13 HTTP/1.1
[Thu Feb 20 04:00:36 2014] [error] [client 127.0.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking_in&date=1392894036&uname=OpenBSD&uptime= 4:00AM  up 18 days, 19:38, 3 users, load averages: 0.26, 0.17, 0.14 HTTP/1.1
[Thu Feb 20 04:00:47 2014] [error] [client 127.0.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking_in&date=1392894047&uname=OpenBSD&uptime= 4:00AM  up 18 days, 19:38, 3 users, load averages: 0.22, 0.17, 0.14 HTTP/1.1
[Thu Feb 20 04:01:26 2014] [error] [client 127.0.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking_in&date=1392894086&uname=OpenBSD&uptime= 4:01AM  up 18 days, 19:39, 3 users, load averages: 0.11, 0.14, 0.13 HTTP/1.1
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 26, near "$var ("
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 31, near "}"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 04:02:03 2014] [error] [client 127.0.0.1] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 26, near "$var ("
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 31, near "}"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 04:02:33 2014] [error] [client 127.0.0.1] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 26, near "$var ("
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 31, near "}"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 04:02:43 2014] [error] [client 127.0.0.1] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 26, near "$var ("
syntax error at /data/serve/ghettoapi/cgi/gapi.pl line 31, near "}"
Execution of /data/serve/ghettoapi/cgi/gapi.pl aborted due to compilation errors.
[Thu Feb 20 04:02:51 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/gapi.pl
[Thu Feb 20 04:03:07 2014] [error] [client 127.0.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking_in&date=1392894187&uname=OpenBSD&uptime= 4:03AM  up 18 days, 19:40, 3 users, load averages: 0.08, 0.13, 0.12 HTTP/1.1
[Thu Feb 20 04:03:34 2014] [error] [client 127.0.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking_in&date=1392894214&uname=OpenBSD&uptime= 4:03AM  up 18 days, 19:41, 3 users, load averages: 0.17, 0.15, 0.13 HTTP/1.1
[Thu Feb 20 04:03:53 2014] [error] [client 172.16.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking_in&date=1392894233&uname=OpenBSD&uptime= 4:03AM  up 18 days, 19:41, 3 users, load averages: 0.19, 0.15, 0.13 HTTP/1.1
[Thu Feb 20 04:04:25 2014] [error] [client 172.16.0.1] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking in HTTP/1.1
[Thu Feb 20 04:08:43 2014] [error] [client 172.16.0.178] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking%20in&hostname=miranda.niko.im&fortune=After living in New York, you trust nobody, but you believe HTTP/1.1
[Thu Feb 20 04:08:53 2014] [error] [client 172.16.0.178] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking%20in&hostname=miranda.niko.im&fortune=Computers are useless.  They can only give you answers. HTTP/1.1
[Thu Feb 20 04:08:54 2014] [error] [client 172.16.0.178] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking%20in&hostname=miranda.niko.im&fortune=Why did the Roman Empire collapse? HTTP/1.1
[Thu Feb 20 04:08:55 2014] [error] [client 172.16.0.178] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking%20in&hostname=miranda.niko.im&fortune=Mustgo, n.: HTTP/1.1
[Thu Feb 20 04:09:01 2014] [error] [client 172.16.0.178] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking%20in&hostname=miranda.niko.im&fortune='For 20 dollars, I'll give you a good fortune next time ...' HTTP/1.1
[Thu Feb 20 04:09:18 2014] [error] [client 172.16.0.178] request failed: erroneous characters after protocol string: GET /cgi/gapi.pl?message=checking%20in&hostname=miranda.niko.im&fortune=Filesystem  512-blocks      Used     Avail Capacity  Mounted on HTTP/1.1
[Thu Feb 20 10:27:42 2014] [notice] SIGHUP received.  Attempting to restart
[Thu Feb 20 10:27:42 2014] [alert] httpd: Could not determine the server's fully qualified domain name, using dnpcor4rtr00.int.ghetto.sh for ServerName
[Thu Feb 20 10:27:42 2014] [notice] Initializing etag from /data/serve/ghettoapi/logs/etag-state
[Thu Feb 20 10:27:42 2014] [notice] Apache configured -- resuming normal operations
[Thu Feb 20 10:27:42 2014] [notice] Accept mutex: sysvsem (Default: sysvsem)
[Thu Feb 20 10:27:52 2014] [error] [client 172.16.0.178] File does not exist: /data/serve/ghettoapi/htdocs/cgi/gapi.pl
[Thu Feb 20 10:41:23 2014] [notice] SIGHUP received.  Attempting to restart
[Thu Feb 20 10:41:23 2014] [alert] httpd: Could not determine the server's fully qualified domain name, using dnpcor4rtr00.int.ghetto.sh for ServerName
[Thu Feb 20 10:41:23 2014] [notice] Initializing etag from /data/serve/ghettoapi/logs/etag-state
[Thu Feb 20 10:41:23 2014] [notice] Apache configured -- resuming normal operations
[Thu Feb 20 10:41:23 2014] [notice] Accept mutex: sysvsem (Default: sysvsem)
[Thu Feb 20 10:41:26 2014] [error] [client 172.16.0.178] script not found or unable to stat: /data/serve/ghettoapi/cgi/cgi
[Thu Feb 20 10:43:45 2014] [error] [client 172.16.0.178] file permissions deny server execution: /data/serve/ghettoapi/cgi/install.conf
Global symbol "$QUERY_STRING" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 6.
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:43:53 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
Global symbol "$QUERY_STRING" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 5.
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:44:02 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
Global symbol "$QUERY_STRING" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 5.
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:44:08 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
syntax error at /data/serve/ghettoapi/cgi/install.conf line 8, near "my "
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:46:58 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
syntax error at /data/serve/ghettoapi/cgi/install.conf line 8, near ")["
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:50:06 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
syntax error at /data/serve/ghettoapi/cgi/install.conf line 8, near ") ("
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:51:40 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
syntax error at /data/serve/ghettoapi/cgi/install.conf line 8, near ") ("
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:51:58 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
syntax error at /data/serve/ghettoapi/cgi/install.conf line 8, near ") ("
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:52:11 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
Bareword found where operator expected at /data/serve/ghettoapi/cgi/install.conf line 9, near "} in"
	(Missing operator before in?)
syntax error at /data/serve/ghettoapi/cgi/install.conf line 9, near "} in "
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:53:23 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
syntax error at /data/serve/ghettoapi/cgi/install.conf line 9, near ")["
syntax error at /data/serve/ghettoapi/cgi/install.conf line 10, near "}"
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:54:16 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (172.16.0.178) at 00:22:fa:43:ae:48 on re0
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Scalar found where operator expected at /data/serve/ghettoapi/cgi/install.conf line 10, near ") 
  $line"
	(Missing operator before $line?)
syntax error at /data/serve/ghettoapi/cgi/install.conf line 10, near ") 
  $line "
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 10:57:12 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (24.8.186.1) at 00:01:5c:63:92:46 on re1
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (24.8.186.1) at 00:01:5c:63:92:46 on re1
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (24.8.186.1) at 00:01:5c:63:92:46 on re1
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (24.8.186.1) at 00:01:5c:63:92:46 on re1
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (24.8.186.1) at 00:01:5c:63:92:46 on re1
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (24.8.186.1) at 00:01:5c:63:92:46 on re1
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Quantifier follows nothing in regex; marked by <-- HERE in m/? <-- HERE  (24.8.186.1) at 00:01:5c:63:92:46 on re1
/ at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$line" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 8.
Global symbol "$name" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$ip" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$tmp1" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$mac" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$tmp2" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$tmp3" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$line" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 9.
Global symbol "$ip" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 10.
Global symbol "$target" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 11.
Global symbol "$mac" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 11.
Global symbol "$target" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 16.
Global symbol "$var" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 19.
Global symbol "$val" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 20.
Global symbol "$var" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 20.
Global symbol "$val" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 21.
Global symbol "$val" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 22.
Global symbol "$var" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 23.
Global symbol "$val" requires explicit package name at /data/serve/ghettoapi/cgi/install.conf line 23.
Execution of /data/serve/ghettoapi/cgi/install.conf aborted due to compilation errors.
[Thu Feb 20 11:09:57 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/install.conf
[Thu Feb 20 12:02:31 2014] [error] [client 172.16.0.178] file permissions deny server execution: /data/serve/ghettoapi/cgi/test.py
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 3, in <module>
    print os.env
AttributeError: 'module' object has no attribute 'env'
[Thu Feb 20 12:02:44 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/test.py
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 5, in <module>
    print os.env
AttributeError: 'module' object has no attribute 'env'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.listdir()
TypeError: listdir() takes exactly 1 argument (0 given)
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.listdir()
TypeError: listdir() takes exactly 1 argument (0 given)
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.listdir()
TypeError: listdir() takes exactly 1 argument (0 given)
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.listdir()
TypeError: listdir() takes exactly 1 argument (0 given)
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environment('QUERY_STRING')
AttributeError: 'module' object has no attribute 'environment'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environment('QUERY_STRING')
AttributeError: 'module' object has no attribute 'environment'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environment('QUERY_STRING')
AttributeError: 'module' object has no attribute 'environment'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environment('QUERY_STRING')
AttributeError: 'module' object has no attribute 'environment'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environment('QUERY_STRING')
AttributeError: 'module' object has no attribute 'environment'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environment('QUERY_STRING')
AttributeError: 'module' object has no attribute 'environment'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environment()
AttributeError: 'module' object has no attribute 'environment'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environ('QUERY_STRING')
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environ('QUERY_STRING')
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 5, in <module>
    print os.environ()
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 6, in <module>
    print os.environ('QUERY_STRING')
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 7, in <module>
    print i, os.environ(i)
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 9, in <module>
    print os.environ('QUERY_STRING')
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 9, in <module>
    print os.environ('QUERY_STRING')
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 9, in <module>
    print os.environ('QUERY_STRING')
AttributeError: _Environ instance has no __call__ method
  File "/data/serve/ghettoapi/cgi/test.py", line 9
    print "-" x 80
              ^
SyntaxError: invalid syntax
[Thu Feb 20 12:14:18 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/test.py
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 11, in <module>
    print os.environ('QUERY_STRING')
AttributeError: _Environ instance has no __call__ method
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 11, in <module>
    print os.environ(QUERY_STRING)
NameError: name 'QUERY_STRING' is not defined
  File "/data/serve/ghettoapi/cgi/test.py", line 14
    
    ^
SyntaxError: invalid syntax
[Thu Feb 20 12:14:49 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/test.py
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 12, in <module>
    for param in QUERY_STIRNG.split('&'):
NameError: name 'QUERY_STIRNG' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 11, in <module>
    for param in QUERY_STIRNG.split('&'):
NameError: name 'QUERY_STIRNG' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 11, in <module>
    for param in QUERY_STIRNG:
NameError: name 'QUERY_STIRNG' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 20, in <module>
    print system('id')
NameError: name 'system' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 20, in <module>
    print system('id')
NameError: name 'system' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 30, in <module>
    print ok
NameError: name 'ok' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 25, in <module>
    (k,v) = param.split('=')
ValueError: need more than 1 value to unpack
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 25, in <module>
    (k,v) = param.split('=')
ValueError: too many values to unpack
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 25, in <module>
    (k,v) = param.split('=')
ValueError: too many values to unpack
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 26, in <module>
    (k,v) = param.split('=')
ValueError: need more than 1 value to unpack
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 29, in <module>
    (k,v) = param.split('=')
ValueError: too many values to unpack
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/test.py", line 26, in <module>
    print count('=',param)
NameError: name 'count' is not defined
  File "/data/serve/ghettoapi/cgi/test.py", line 26
    if '=' not in param or if len(param.split('=')) > 2:
                            ^
SyntaxError: invalid syntax
[Thu Feb 20 12:35:53 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/test.py
  File "/data/serve/ghettoapi/cgi/test.py", line 26
    if '=' not in param  
                        ^
SyntaxError: invalid syntax
[Thu Feb 20 12:36:18 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/test.py
  File "/data/serve/ghettoapi/cgi/test.py", line 13
    print '''
        ^
IndentationError: expected an indented block
[Thu Feb 20 12:48:47 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/test.py
[Thu Feb 20 12:51:34 2014] [error] [client 172.16.0.178] attempt to invoke directory as script: /data/serve/ghettoapi/cgi
[Thu Feb 20 12:51:57 2014] [error] [client 172.16.0.178] attempt to invoke directory as script: /data/serve/ghettoapi/cgi
[Thu Feb 20 12:51:57 2014] [error] [client 172.16.0.178] script not found or unable to stat: /data/serve/ghettoapi/cgi/test.py
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 87, in <module>
    VALUES (%s,%s,%s,%s)''' % (message,date,hostname,uptime)
NameError: name 'message' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 87, in <module>
    VALUES (%s,%s,%s,%s)''' % (message,date,hostname,uptime)
NameError: name 'message' is not defined
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 87, in <module>
    VALUES (%s,%s,%s,%s)''' % (message,date,hostname,uptime)
NameError: name 'message' is not defined
  File "/data/serve/ghettoapi/cgi/checkin", line 90
    params['hostname,
                    ^
SyntaxError: EOL while scanning string literal
[Thu Feb 20 13:02:15 2014] [error] [client 172.16.0.178] Premature end of script headers: /data/serve/ghettoapi/cgi/checkin
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 92, in <module>
    params['ip'] )
TypeError: not all arguments converted during string formatting
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 96, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: attempt to write a readonly database
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 96, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: attempt to write a readonly database
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 96, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 96, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 96, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 96, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 13, in <module>
    db = sqlite3.connect('/ghetto.db')
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 13, in <module>
    db = sqlite3.connect('/cgi/ghetto.db')
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 97, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 97, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 94, in <module>
    params['hostname'],
KeyError: 'hostname'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 100, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 100, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 93, in <module>
    params['date'],
KeyError: 'date'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 100, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 100, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 101, in <module>
    cursor.commit()
AttributeError: 'sqlite3.Cursor' object has no attribute 'commit'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 101, in <module>
    cursor.committ()
AttributeError: 'sqlite3.Cursor' object has no attribute 'committ'
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 101, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: unable to open database file
Traceback (most recent call last):
  File "/data/serve/ghettoapi/cgi/checkin", line 98, in <module>
    cursor.execute(sql)
sqlite3.OperationalError: no such table: events
[Thu Feb 20 14:17:35 2014] [error] [client 172.16.0.178] file permissions deny server execution: /data/serve/ghettoapi/cgi/ghetto.db
[Thu Feb 20 14:21:07 2014] [notice] caught SIGTERM, shutting down
[Thu Feb 20 14:21:09 2014] [alert] httpd: Could not determine the server's fully qualified domain name, using dnpcor4rtr00.int.ghetto.sh for ServerName
[Thu Feb 20 14:21:09 2014] [notice] Initializing etag from /data/serve/ghettoapi/logs/etag-state
[Thu Feb 20 14:21:09 2014] [notice] Apache configured -- resuming normal operations
[Thu Feb 20 14:21:09 2014] [notice] Accept mutex: sysvsem (Default: sysvsem)
[Thu Feb 20 14:21:17 2014] [error] [client 172.16.0.178] file permissions deny server execution: /data/serve/ghettoapi/cgi/ghetto.db
[Thu Feb 20 14:21:19 2014] [error] [client 172.16.0.178] attempt to invoke directory as script: /data/serve/ghettoapi/cgi
[Thu Feb 20 14:22:44 2014] [notice] caught SIGTERM, shutting down
[Thu Feb 20 14:22:44 2014] [alert] httpd: Could not determine the server's fully qualified domain name, using dnpcor4rtr00.int.ghetto.sh for ServerName
[Thu Feb 20 14:22:44 2014] [notice] Initializing etag from /data/serve/ghettoapi/logs/etag-state
[Thu Feb 20 14:22:44 2014] [notice] Apache configured -- resuming normal operations
[Thu Feb 20 14:22:44 2014] [notice] Accept mutex: sysvsem (Default: sysvsem)
[Thu Feb 20 14:23:30 2014] [notice] caught SIGTERM, shutting down
[Thu Feb 20 14:23:30 2014] [alert] httpd: Could not determine the server's fully qualified domain name, using dnpcor4rtr00.int.ghetto.sh for ServerName
[Thu Feb 20 14:23:30 2014] [notice] Initializing etag from /data/serve/ghettoapi/logs/etag-state
[Thu Feb 20 14:23:30 2014] [notice] Apache configured -- resuming normal operations
[Thu Feb 20 14:23:30 2014] [notice] Accept mutex: sysvsem (Default: sysvsem)
[Thu Feb 20 14:24:34 2014] [error] [client 172.16.0.178] file permissions deny server execution: /data/serve/ghettoapi/cgi/ghetto.db
